config.example/helm/ingress.yml

```diff
@@ -7,9 +7,11 @@
 controller:
   # Use host network to listen on ports 80 and 443
   hostNetwork: true
+  # Service type LoadBalancer requires a load balancer to be configured, e.g.
+  # MetalLB in an on-prem cluster. See metallb.yml for a sample definition.
   # Use NodePort for on-prem where we don't have a load balancer
   service:
-    type: NodePort
+    type: LoadBalancer
```

Comment:
I think this is a good change, but we're going to need to update the docs to show that the LB is required before bringing up the ingress controller.